ads/sovrn.md

```diff
@@ -0,0 +1,50 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# sovrn
```

Comment:
We followed the pattern of the other .md files and have the following included. 

> For semantics of configuration, please see ad network documentation.

Is this a new requirement to have an actual link or email address for publishers to use?
We can certainly do that just need  a little clarification on if you are looking for a link to full blown documentation or just an email etc.